frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -714,19 +762,44 @@ const Chart = (props: ChartProp) => {
                         y: map ? undefined : transform(yaxis, "top")(evt?.clientY),
                         lon: map ? xaxis.p2c() : undefined,
                         x: map ? undefined : transform(xaxis, "left")(evt?.clientX),
-                    })
-                )
+                    }),
+                ),
             );
         },
-        [dispatch, module, id, onClick]
+        [dispatch, module, id, onClick],
     );
 
-    const onInitialized = useCallback(
-        (figure: Readonly<Figure>, graphDiv: Readonly<HTMLElement>) => {
-            onClick && graphDiv.addEventListener("click", clickHandler);
-        },
-        [onClick, clickHandler]
-    );
+    const onInitialized = useCallback((figure: Readonly<Figure>, graphDiv: Readonly<HTMLElement>) => {
+        onClick && graphDiv.addEventListener("click", clickHandler);
+        plotRef.current = graphDiv as HTMLDivElement;
+
+        if (animationDataValues) {
+            const runAnimation = async () => {
+                if (plotRef.current && frames?.to?.data && frames.to.traces.length > 0) {
+                    await Plotly.animate(
+                        plotRef.current,
+                        {
+                            data: frames.to.data,
+                            traces: frames.to.traces,
+                            layout: layout,
+                        },
+                        {
```

Comment:
You may want to move this dict to some 'default' constant value. 
Then we could modify it with the setting of the *animation* prop.
At least the default values don't show up in the middle of the code.
(and reuse it later)